qutebrowser/browser/webkit/network/filescheme.py

```diff
@@ -77,7 +77,7 @@ def parent_dir(directory):
     Return:
         The path to the parent directory.
     """
-    return os.path.normpath(os.path.join(directory, os.pardir))
+    return str(pathlib.Path(directory / pathlib.Path(directory).parent))
```

Comment:
this isn't making sense to me. Shouldn't it just be `str(pathlib.Path(directory).parent)`? We probably don't even need a helper any more.